tests/testutils.py

```
@@ -32,19 +32,17 @@
 # pylint: enable=wrong-import-position
 sys.path.pop(0)
 
-IMPORT_ENCHANT_SUCCESSFUL = False
 try:
     import enchant # type: ignore
     IMPORT_ENCHANT_SUCCESSFUL = True
 except (ImportError,):
```

Ruff (0.15.10)

[warning] A length-one tuple literal is redundant in exception handlers

Replace with `except ImportError`

(B013)